examples/experimental/wsl2.yaml

```diff
@@ -0,0 +1,16 @@
+# This template requires Lima v0.18.0 or later and only works on Windows versions
+# that support WSL2 (Windows 10 Build >= 19041, all Windows 11).
+vmType: wsl2
+
+images:
+# Source: https://github.com/runfinch/finch-core/blob/main/Dockerfile
+- location: "https://deps.runfinch.com/common/x86-64/finch-rootfs-production-amd64-1690920103.tar.zst"
+  arch: "x86_64"
+  digest: "sha256:53f2e329b8da0f6a25e025d1f6cc262ae228402ba615ad095739b2f0ec6babc9"
+
+mountType: wsl2 
+
+# Use system because of an error when setting up RootlessKit (see )
```

Comment:
see ?

Reply:
Forgot link, fixed in latest revision. Also added docs